src/software/ai/navigator/path_planner/hrvo/velocity_obstacle.cpp

```diff
@@ -49,21 +49,21 @@ VelocityObstacle VelocityObstacle::generateVelocityObstacle(
     const Vector robot_to_obstacle_vector        = obstacle.origin() - robot.origin();
     const Angle robot_relative_to_obstacle_angle = robot_to_obstacle_vector.orientation();
 
-    // opening angle of each side relative = arcsin((rad_A + rad_B) / distance)
-    Angle opening_angle = Angle::asin((robot.radius() + obstacle.radius()) /
-                                      robot_to_obstacle_vector.length());
+    // The robot is colliding with obstacle.
+    // Creates Velocity Obstacle with the sides being 180 degrees
+    // apart from each other (90 degrees relative to the robot to
+    // obstacle vector) with center being the center of obstacle.
+    // Subtracting by a slight offset to avoid the velocity obstacle
+    // from being the complement of what we want (Since VOs are defined as
+    // area created between the smallest angles between the two sides.
+    Angle opening_angle = Angle::quarter() - Angle::fromRadians(0.0001);
 
-    if (robot_to_obstacle_vector.lengthSquared() <
+    if (robot_to_obstacle_vector.lengthSquared() >
```

Comment:
Whats the purpose of this change?

Reply:
it's meant to avoid a sigfpe in the asin when (robot.radius() + obstacle.radius()) is greater than robot_to_obstacle_vector.length(). I guess technically, it should be >=, but it doesn't really matter I don't think